scripts/development/.env.example

```diff
@@ -42,10 +42,10 @@ EMAIL_HOST=email-service
 EMAIL_FROM="Trubudget Notification Service"
 EMAIL_SUBJECT="Trubudget Notificaiton"
 EMAIL_TEXT="You have received a notification."
-AUTHENTICATION=none
+#AUTHENTICATION=none
 ACCESS_CONTROL_ALLOW_ORIGIN=*
 EMAIL_LOG_LEVEL=info
-#AUTHENTICATION=JWT
+AUTHENTICATION=JWT
```

Comment:
email service should work with AUTHENTICATION=none. If it doesn't, it's a bug. Should be fixed by https://github.com/openkfw/TruBudget/pull/1818

Reply:
@SamuelPull I reverted the change